interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca

```diff
@@ -2,6 +2,8 @@ tosca_definitions_version: tosca_simple_yaml_1_3
 interface_types:
   tosca.interfaces.relationship.Configure:
     derived_from: tosca.interfaces.Root
+	 metadata:
```

Comment:
`metadata` on the same level as `derived_from`

Reply:
fixed